core/templates/app.constants.ts

```diff
@@ -189,6 +189,8 @@ export const AppConstants = {
   AUDIO_UPLOAD_URL_TEMPLATE: '/createhandler/audioupload/<exploration_id>',
   IMAGE_UPLOAD_URL_TEMPLATE: (
     '/createhandler/imageupload/<entity_type>/<entity_id>'),
+  PROFILE_PHOTO_UPLOAD_URL_TEMPLATE: (
+    '/profile_photo_upload/<username>'),
```

Comment:
I'm not sure what the backend call will look like. We need to change this once we know!

Reply:
TODO: Need to remove this `PROFILE_PHOTO_UPLOAD_URL_TEMPLATE` I created and change the functionality to use the `PreferencesHandler`.